### DISCUSSION TITLE
Partially link a package that is partially conflict with another package?

### DISCUSSION THREAD

**Discussion by tddschn** [Homebrew/discussions#3187] [Tap maintenance and Homebrew development] [answered]
Formula `moreutils` is conflict with formula `parallel` because both provides executable `parallel`.

IMO, `moreutils` can be seen as partially conflict with `parallel` since it provides a lot more executables.

For my use case, I want to use `parallel` from `parallel` while have all the other tools from `moreutils` installed.

I want to do this with a command like
```
$ brew install parallel
$ brew install moreutils --no-link parallel
```
Do you think it's a good idea to support this feature?

**Comment by SMillerDev** [comment, marked as answer]
You can just do 

```
$ brew install parallel
$ brew install moreutils
$ brew link --force parallel
```
